### DISCUSSION TITLE
When generate docker-compose aspire dashbord not shown frontend apps (react, vue, angular)

### DISCUSSION THREAD

**Discussion by timeshift92** [microsoft/aspire#4305] [Q&A]
Hello I'm using docker swarm as orchestrator, and i have use in docker swarm aspire. but I got mistakes. if i generate docker compose
[#203](https://github.com/prom3theu5/aspirational-manifests/issues/205)

frontend service not show in aspire dashboard, help me please.

**Comment by davidfowl** [comment]
It's unclear what exactly the problem you're experiencing is. The standalone aspire dashboard does not support showing the list of resources (if that's what you mean). Can you clarify what you are doing and what you expect to see and what you actually see?

**Comment by timeshift92** [comment]
okay one moment. i show images


![image](https://github.com/prom3theu5/aspirational-manifests/assets/13614530/656e03cb-54d7-43bf-9aaa-de50baac85cb)
![image](https://github.com/prom3theu5/aspirational-manifests/assets/13614530/1b072f59-a472-423e-985d-49a212f224cb)

in aspire dashboard you can see all aspire services which you run with aspire. but if you run
https://github.com/dotnet/aspire-samples/tree/main/samples/AspireWithJavaScript
this example and generate docker-compose you can't see react and angular and vue services in aspire dashboard

**Reply by davidfowl** [reply]
That's by design. 

https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/standalone

> The dashboard also has functionality for viewing .NET Aspire resources. The dashboard resource features are disabled when it is run in standalone mode. To enable the resources UI, [add configuration for a resource service](https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/configuration#resources).

When you run outside of the aspire orchestrator, there needs to be another service that tells the dashboard what the list of resources are. There's none that exists at the moment outside of local development and azure container apps. The protocol is open though so one could be built for other environments.

Here's the grpc server you have to implement https://github.com/dotnet/aspire/blob/6d41a31700c10250f5d33154c9556d1f52b5a2cb/src/Aspire.Hosting/Dashboard/proto/resource_service.proto

**Comment by timeshift92** [comment]
see. thanks for information. well if i need use another .net family service we need implement this feature
